src/opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.cpp

```diff
@@ -71,6 +71,13 @@ initFromState(const EclipseState& eclState, const Schedule&)
 
     brineReferenceDensity_[regionIdx] = Brine::liquidDensity(T_ref, P_ref, salinity_[regionIdx], extrapolate);
     co2ReferenceDensity_[regionIdx] = CO2::gasDensity(T_ref, P_ref, extrapolate);
+
+    OpmLog::info("CO2STORE is enabled. \n The surface density of CO2 is  " + std::to_string(co2ReferenceDensity_[regionIdx])
+                 + "kg/m3 \n The surface density of Brine is  " + std::to_string(brineReferenceDensity_[regionIdx])
+                 + "kg/m3"
+                 + "\n The surface densities are computed using the reference pressure ( " + std::to_string(P_ref) 
+                 + "Pa) and the reference temperature (" +  std::to_string(T_ref) + "K)." 
```

Comment:
Do we not anticipate anyone running `CO2STORE` cases in unit conventions other than METRIC?

Reply:
> For simplicity I just dumped everything in SI units. I don't think the need is urgent, but it shouldn't be too hard to convert using the `from_si` functions.

Right.  You'll probably need the [`name()`](https://github.com/OPM/opm-common/blob/980ac2599acf475129019e8de8a352b05d8b280d/opm/input/eclipse/Units/UnitSystem.hpp#L125) function to retrieve the associate unit strings as well, but that's easy enough.